src/Lab/VulkanTriangle/HelloTriangleApplication.cs

```diff
@@ -636,7 +636,7 @@ private Extent2D ChooseSwapExtent(SurfaceCapabilitiesKHR capabilities)
             }
 
             var actualExtent = new Extent2D
-            { Height = (uint) _window.Size.Height, Width = (uint) _window.Size.Width };
+            { Height = (uint) _window.Size.Y, Width = (uint) _window.Size.X };
```

Comment:
what's Extent2D and why do we need it here?

Reply:
VkRect2D is comprised of int points and uint dimensions as discussed.